File: janari0_mobile/lib/model/payment.dart
import 'package:json_annotation/json_annotation.dart';

part 'payment.g.dart';

@JsonSerializable()
class Payment {
  String? paymentMethodNonce;
  double? amount;
  int? buyerId;
  bool? successful;
  int? productSaleId;

  Payment();

  factory Payment.fromJson(Map<String, dynamic> json) =>
      _$PaymentFromJson(json);

  /// Connect the generated [_$PaymentToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$PaymentToJson(this);
}

File: Janari0.Model/Payment.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Janari0.Model
{
    public class Payment
    {
        public string PaymentMethodNonce { get; set; }
        public decimal Amount { get; set; }
        public bool Successful { get; set; }
        public int ProductSaleId { get; set; }
        public int BuyerId { get; set; }
    }
}
